src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/EligibilityServiceTest.kt

```diff
@@ -213,7 +213,7 @@ class EligibilityServiceTest {
         conditionalReleaseDate = null,
       ),
     )
-    assertThat(result).containsExactly("has no conditional release date")
+    assertThat(result).containsExactly("has no conditional release date", "release date in past")
```

Comment:
I think this is a touch misleading as in the case that they have no ARD or CRD, we'd still get the second reason of 'release date in past', but I suppose that's somewhat unavoidable if we're checking all potential exclusion reasons at once rather than having a 1-by-1 hierarchy

Reply:
I think this ties in with Andy's comment re having an additional rule checking for `noReleaseDate` which we can add later.

Reply:
Ah, yep, missed that!